packages/x-components/CHANGELOG.md

```diff
@@ -1,29 +1,28 @@
 # Change Log
```

Comment:
I thin that updating the branch with main, it will solve the changes in this file